modules/dgi_migrate_foxml_standard_mods/migrations/dgis_nodes.yml

```
@@ -1973,6 +2003,455 @@ process:
+      callable: array_filter
+    - plugin: null_coalesce
+  field_publication_genre:
+    - <<: *base_mods_node
+      query: 'mods:relatedItem[@type="host"]/mods:genre'
+    - plugin: callback
+      callable: iterator_to_array
+    - plugin: skip_on_empty
+      method: process
+    - plugin: multiple_values
+    - plugin: dgi_migrate.sub_process
+      process_values: true
+      values:
+        _vid:
+          - plugin: default_value
+            default_value: genre
+        <<: *generic_term_after
+    - <<: *generic_term_extract
+  field_extent_total_pages:
+    - <<: *base_mods_node
+      query: 'mods:part/mods:extent/mods:total'
+    - plugin: callback
+      callable: iterator_to_array
+    - plugin: multiple_values
+    - plugin: dgi_migrate.subproperty
+      property: nodeValue
+    - plugin: single_value
+    - plugin: callback
+      callable: array_filter
+    - plugin: null_coalesce
+  field_extent_first_page:
+    - <<: *base_mods_node
+      query: 'mods:part/mods:extent/mods:start'
+    - plugin: callback
+      callable: iterator_to_array
+    - plugin: multiple_values
+    - plugin: dgi_migrate.subproperty
+      property: nodeValue
+    - plugin: single_value
+    - plugin: callback
+      callable: array_filter
+    - plugin: null_coalesce
+  field_extent_last_page:
+    - <<: *base_mods_node
+      query: 'mods:part/mods:extent/mods:end'
+    - plugin: callback
+      callable: iterator_to_array
+    - plugin: multiple_values
+    - plugin: dgi_migrate.subproperty
+      property: nodeValue
+    - plugin: single_value
+    - plugin: callback
+      callable: array_filter
+    - plugin: null_coalesce
+  field_admin_note:
+    - <<: *base_mods_node
+      query: 'mods:note[@type="admin"]'
+    - plugin: callback
+      callable: iterator_to_array
+    - plugin: multiple_values
+    - plugin: dgi_migrate.subproperty
+      property: nodeValue
+    - plugin: single_value
+    - plugin: callback
+      callable: array_filter
+    - plugin: null_coalesce
+  field_funder:
+    - <<: *base_mods_node
+      query: 'mods:role/mods:roleTerm="funder"'
+    - plugin: callback
+      callable: iterator_to_array
+    - plugin: skip_on_empty
+      method: process
+    - plugin: multiple_values
+    - plugin: dgi_migrate.sub_process
+      process_values: true
+      values:
+        _vid:
+          - plugin: default_value
+            default_value: funders
+        <<: *generic_term_after
+    - <<: *generic_term_extract
+  field_sponsorship_information:
+    - <<: *base_mods_node
+      query: 'mods:note[@type="funding"]'
+    - plugin: callback
+      callable: iterator_to_array
+    - plugin: multiple_values
+    - plugin: dgi_migrate.subproperty
+      property: nodeValue
+    - plugin: single_value
+    - plugin: callback
+      callable: array_filter
+    - plugin: null_coalesce
+  _use_license:
+    - <<: *base_mods_node
+      query: 'mods:accessCondition[@type="use and reproduction"][@displayLabe="Creative Commons"]'
```

I know it's such in the spec, but, seems like a typo?:

```suggestion
      query: 'mods:accessCondition[@type="use and reproduction"][@displayLabel="Creative Commons"]'
```

---

Made it so in #135